tests/unit/gpkg/gpkg.spec.ts

```diff
@@ -40,15 +40,15 @@ describe('gpkg', () => {
     jest.mock('better-sqlite3');
```

Comment:
does it actually works here?
for me mocking modules only worked in global scoop

Reply:
בס"ד
The module was already mocked in the global scope, you're right